requirements/heroku.txt

```diff
@@ -1,6 +1,6 @@
 -r base.txt
 # Additional dependencies for Heroku deployment
-dj-database-url
-gunicorn
-psycopg2
-whitenoise
+dj-database-url==0.4.1
+gunicorn==19.6.0
+psycopg2==2.6.2
+whitenoise==3.2.2
```

Comment:
Does the presence of whitenoise here mean Heroku users will not be serving media files via apache or nginx? That's very cool actually.